jetty-start/src/main/java/org/eclipse/jetty/start/BaseBuilder.java

```diff
@@ -222,72 +227,98 @@ public boolean accept(Path entry) throws IOException
             }
         }
 
-        if (!newlyAdded.isEmpty())
+        if ((startArgs.isCreateStartD() && (!Files.exists(startd) || Files.exists(startini))) ||
+            (!newlyAdded.isEmpty() && !Files.exists(startini) && !Files.exists(startd)))
```

Comment:
So if you put both --create-startd and --create-start-ini on the command line what would happen? Is it dependent on the ordering on the command line? Should it be an error?

Reply:
No it will first create start.ini and then create start.d and move that start.ini into it.